docs/components/component_overview_building_blocks_list.js

```diff
@@ -0,0 +1,99 @@
+// @flow strict
```

Comment:
What's with the snake_case on this (and other) files? All our other files are either camelCase or PascalCase (aside from how Next handles spaces in page names) — these files should match our convention

Reply:
Also: is there a reason for these to be split out to separate files? These consts could all live in the same file.

Reply:
they were inside /pages  where we do that. Moved out cos Next didn't allow those files in /pages. Kept the naming convention.

<img width="311" alt="Screen Shot 2022-03-24 at 5 56 26 PM" src="https://user-images.githubusercontent.com/10593890/160016980-3898ef72-997e-4d9b-9f7f-7fca5a7e4246.png">

Reply:
DONE